modules/logging-clusterlogging-cr-config.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * logging/logging-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="logging-clusterlogging-cr-config_{context}"]
+= Configuring the ClusterLogging CR
+
+You can adjust the configuration of the {logging} components by editing the `ClusterLogging` CR.
+
+.Procedure
+
+. Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:
++
+[source,terminal]
+----
+$ oc -n openshift-logging edit ClusterLogging instance
+----
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
```

Comment:
Full field reference should be generated as per @jcantrill s work, this should be a more minimal example of key fields that often need to be tweaked.

Until vector and loki are default, we should also show how to enable them, e.g.:
```
apiVersion: logging.openshift.io/v1
kind: ClusterLogging
metadata:
  name: instance
  namespace: openshift-logging
spec:
  managementState: Managed
  collection:
    type: vector
  logStore:
     type: lokistack
     lokistack:
       name: logging-loki
```
 For loki we also need to explain how to set up or use an S3 store, @periklis is the experrt.